functions/idvtoauth0/authzero.py

```diff
@@ -63,7 +63,7 @@ def __init__(self, config):
             level=logging.INFO
         )
 
-        self.logger = logging.getLogger('CISAuthZero')
+        self.logger = logging.getLogger('cis-idvtoauth0')
```

Comment:
I liked your CaMel casing better.  but let's go with it.

Reply:
used the same name as the class in main.py basically